cypress/elements/fileMenu/index.js

```diff
@@ -1,57 +1,54 @@
-/*  FIXME:
-    Most of the element selection in this file is quite poor because 
-    of the FileMenu being old and still uses MUI.
-    This can be refactored to use proper data-test selectors once the new
-    FileMenu component is in place.
-*/
-
 import { clickMenuBarFileButton } from '../menuBar'
 
-const deleteModalEl = '*[class^="MuiDialog-container"]'
-const fileMenuItemEl = '*[role="menuitem"]'
+const deleteModalEl = 'file-menu-delete-modal'
+const fileMenuItemEl = 'file-menu-container'
 
 export const FILE_MENU_BUTTON_NEW = 'New'
-export const FILE_MENU_BUTTON_OPEN = 'Open'
-export const FILE_MENU_BUTTON_SAVE = 'Save'
-export const FILE_MENU_BUTTON_SAVEAS = 'Save as...'
-export const FILE_MENU_BUTTON_RENAME = 'Rename'
-export const FILE_MENU_BUTTON_TRANSLATE = 'Translate'
-export const FILE_MENU_BUTTON_SHARE = 'Share'
-export const FILE_MENU_BUTTON_GETLINK = 'Get link'
+export const FILE_MENU_BUTTON_OPEN = 'Open…'
+export const FILE_MENU_BUTTON_SAVE_EXISTING = 'Save'
+export const FILE_MENU_BUTTON_SAVE_NEW = 'Save…'
+export const FILE_MENU_BUTTON_SAVEAS = 'Save as…'
+export const FILE_MENU_BUTTON_RENAME = 'Rename…'
+export const FILE_MENU_BUTTON_TRANSLATE = 'Translate…'
+export const FILE_MENU_BUTTON_SHARE = 'Share…'
+export const FILE_MENU_BUTTON_GETLINK = 'Get link…'
 export const FILE_MENU_BUTTON_DELETE = 'Delete'
 
 export const closeFileMenu = () =>
-    cy
-        .get(fileMenuItemEl)
+    cy.get('[data-test="file-menu-toggle-layer"]').click('topLeft')
```

Comment:
Why didn't this get a `const` at the top of the file and why not use `getBySel`?

Reply:
Fixed.